### ISSUE TITLE
tabbyml 0.14 - gitlab fetch issues failed - data from JSON: invalid type: null, expected a string

### ISSUE THREAD

**Issue by ifelsefi** [TabbyML/tabby#2724]
```2024-07-24T11:06:04.990162-05:00 [INFO]: Finished garbage collection for code index: 4164 items kept, 0 items removed
2024-07-24T11:06:05.012636-05:00 [INFO]: Indexing documents for repository development/repo
2024-07-24T11:06:05.602745-05:00 [ERROR]: Failed to fetch issues: could not parse alloc::vec::Vec<tabby_webserver::service::background_job::third_party_integration::issues::GitlabIssue> data from JSON: invalid type: null, expected a string
2024-07-24T11:06:05.602808-05:00 [INFO]: 0 documents has been processed, 0 has been updated
2024-07-24T11:06:05.639969-05:00 [INFO]: Job completed successfully```

**Comment by wsxiaoys** [comment]
Thanks for reporting the issue, this is indeed a bug - `description` might be null in gitlab api responses, fixing in #2729

**Comment by wsxiaoys** [comment]
Released in 0.15
